src/Apps/Order/Routes/Offer/index.tsx

```diff
@@ -43,52 +37,49 @@ export interface OfferProps extends SystemContextProps {
   isCommittingMutation: boolean
 }
 
-export interface OfferState {
-  offerValue: number
-  offerNoteValue: TextAreaChange
-  formIsDirty: boolean
-  lowSpeedBumpEncountered: boolean
-  isToggleRadio: boolean
-}
+export const OfferRoute: FC<OfferRouteProps> = ({
+  order,
+  router,
+  dialog,
+  commitMutation,
+  isCommittingMutation,
+}) => {
+  const { trackEvent } = useTracking()
 
-const logger = createLogger("Order/Routes/Offer/index.tsx")
+  const [formIsDirty, setFormIsDirty] = useState(false)
+  const [lowSpeedBumpEncountered, setLowSpeedBumpEncountered] = useState(false)
+  const [offerNoteValue, setOfferNoteValue] = useState({
+    exceedsCharacterLimit: false,
+    value: "",
+  })
+  const [offerValue, setOfferValue] = useState(0)
 
-@track()
-export class OfferRoute extends Component<OfferProps, OfferState> {
-  state: OfferState = {
-    formIsDirty: false,
-    lowSpeedBumpEncountered: false,
-    offerNoteValue: { exceedsCharacterLimit: false, value: "" },
-    offerValue: 0,
-    isToggleRadio: false,
```

Comment:
I got rid of `isToggleRadio` as it doesn't seem to be used anywhere, please ⏰  if I'm wrong.